aiphysim/dataloading/dataloader_spacetime.py

```diff
@@ -117,58 +101,44 @@ def __init__(
                 axis=0,
             )
 
-            training_data = np.stack(
-                [
-                    hdata["tasks"]["p"][: int(nt * train_size)],
-                    hdata["tasks"]["T"][: int(nt * train_size)],
-                    hdata["tasks"]["u"][: int(nt * train_size)],
-                    hdata["tasks"]["w"][: int(nt * train_size)],
-                ],
-                axis=0,
-            )
+            self.data = self.data.transpose(1, 0, 3, 2)  # [t, c, z, x]
 
-            self._mean = np.mean(training_data, axis=(0, 2, 3))
-            self._std = np.std(training_data, axis=(0, 2, 3))
-
-        self.data = self.data.astype(np.float32)
```

Comment:
Why are you changing the data type?

Reply:
It was redundant, I already set it to float (which is float32 in pytorch) in the __getitem__